### ISSUE TITLE
phoneinfoga not working in india

### ISSUE THREAD

**Issue by kingluseth** [sundowndev/PhoneInfoga#1426]
Issue is that all urls are not working and just lead to blank pages i have tried dozen of no.s in india who i know have social media accounts yet google the urls do not work
 
I am using kali linux and phoneinfoga version 2.11.

**Comment by kingluseth** [comment]
![Screenshot_2024-08-18_18_20_17](https://github.com/user-attachments/assets/f26e0f84-e058-4902-9811-145742b67af2)

**Comment by pierceparks** [comment]
Yes this is true. I had same experience at a period when I had no option than to go hire a hacker online to get it fixed in within 1 hour. Good enough i met a professional that got the job well done. These hackers are best with quality services. hackerspytech @ gmail. com .I have been using them and recommended to friends as well.

**Comment by Satish2421** [comment]
> ![Screenshot_2024-08-18_18_20_17](https://github.com/user-attachments/assets/f26e0f84-e058-4902-9811-145742b67af2)

how to solve this problem and did u get any solutions
